models/Document/DocType.php

```diff
@@ -99,6 +99,13 @@ class DocType extends Model\AbstractModel
      */
     protected $modificationDate;
 
+    /**
+     * @internal
+     *
+     * @var null|int
+     */
+    protected $staticGeneratorEnabled;
```

Comment:
Why is this an integer and not an boolean?

Reply:
I think that was by mistake, but hard to fix now, without a BC break, I'd say that's a topic for 11 then.

Reply:
#10819 made an issue so we don't forget 😊